apps/scan/backend/src/store.ts

```diff
@@ -823,15 +827,44 @@ export class Store {
     return safeParseSystemSettings(result.data).unsafeUnwrap();
   }
 
+  isContinuousExportOperationInProgress(): boolean {
+    const row = this.client.one(
+      `
+      select
+        is_continuous_export_operation_in_progress as isContinuousExportOperationInProgress
+      from is_continuous_export_operation_in_progress
+      `
+    ) as { isContinuousExportOperationInProgress: number } | undefined;
+    return Boolean(row?.isContinuousExportOperationInProgress);
```

Comment:
I just added a type + basic utils to `admin/backend/src/store.ts` for being more explicit about these boolean conversions. What do you think about copying those or moving them to a shared library to use in this `store` file?

Reply:
Oh nice! Can move to a shared location

Reply:
✅ https://github.com/votingworks/vxsuite/pull/4213/commits/a9caa1408ae3d1b3c36cc007074922209d75f28c